### DISCUSSION TITLE
Tracing Celebrity Locations – Angelina Jolie & Jonny Lee Miller in Dumbo

### DISCUSSION THREAD

**Discussion by giscus** [neospl0it/neospl0it.github.io#7] [Comments]
# Tracing Celebrity Locations – Angelina Jolie & Jonny Lee Miller in Dumbo

Explore how OSINT techniques were applied to locate Jonny Lee Miller’s apartment in Dumbo, Brooklyn, following Angelina Jolie’s visit. This case study uses geospatial tools and visual analysis to responsibly uncover insights.

https://neospl0it.github.io/posts/tracking-celebrity-location-geo-osint/

**Comment by neospl0it** [comment]
{Theme Test}(Light)
